entry_types/paged/packages/pageflow-paged/src/frontend/ChapterFilter.js

```diff
@@ -1,6 +1,8 @@
 import _ from 'underscore';
+import {Object} from 'pageflow/frontend'
+import {state} from './state';
 
-pageflow.ChapterFilter = pageflow.Object.extend({
+export const ChapterFilter = Object.extend({
```

Comment:
Just as a quick note while taking a glance at the PR: Rewriting `pageflow-paged/frontend` to import/export is not strictly required for this task, right? Only the stuff that needs to be reused in `pageflow/frontend` needs to be exported so `pageflow-paged/frontend` can import. `pageflow-paged/frontend` will still need to define the global to be compatible with existing Pageflow plugins. It can do this by using import/export internally and then defining the global in one place. It also can also keep defining things on the global in individual files. This would mean significantly less changes to test and review and might make getting specs to run much easier,

Reply:
The approach i took is defining import/export internally and then defining the global in one place. so in the end all that gets extended to pageflow global.

Reply:
This definitely the nicer/more modern solution. Just wanted to point out the alternative in case it's less effort.

Reply:
Yeah you are right that would have been less time consuming to selectively import export specific files